halo2_frontend/src/dev/cost.rs

```diff
@@ -555,6 +555,475 @@ mod tests {
+                        let (_, abcd3) = config
+                            .assign_gate(&mut region, &mut offset, None, [9, 2, 1, 1])
+                            .expect("todo");
+                        offset += 1;
+                        region
+                            .constrain_equal(abcd1[1].cell(), abcd1[3].cell())
+                            .expect("todo");
+                        region
+                            .constrain_equal(abcd2[1].cell(), abcd1[3].cell())
+                            .expect("todo");
+                        region
+                            .constrain_equal(abcd3[0].cell(), abcd1[3].cell())
+                            .expect("todo");
+
+                        // Enable a dynamic lookup (powers of two)
+                        let table: Vec<_> =
+                            (0u64..=10).map(|exp| (exp, 2u64.pow(exp as u32))).collect();
+                        let lookups = [(2, 4), (2, 4), (10, 1024), (0, 1), (2, 4)];
+                        for (table_row, lookup_row) in table
+                            .iter()
+                            .zip(lookups.iter().chain(std::iter::repeat(&(0, 1))))
+                        {
+                            region
+                                .assign_fixed(
+                                    || "",
+                                    config.s_lookup,
+                                    offset,
+                                    || Value::known(F::ONE),
+                                )
+                                .expect("todo");
+                            region
+                                .assign_fixed(
+                                    || "",
+                                    config.s_ltable,
+                                    offset,
+                                    || Value::known(F::ONE),
+                                )
+                                .expect("todo");
+                            let lookup_row0 = Value::known(F::from(lookup_row.0));
+                            let lookup_row1 = Value::known(F::from(lookup_row.1));
+                            region
+                                .assign_advice(|| "", config.a, offset, || lookup_row0)
+                                .expect("todo");
+                            region
+                                .assign_advice(|| "", config.b, offset, || lookup_row1)
+                                .expect("todo");
+                            let table_row0 = Value::known(F::from(table_row.0));
+                            let table_row1 = Value::known(F::from(table_row.1));
+                            region
+                                .assign_fixed(|| "", config.d, offset, || table_row0)
+                                .expect("todo");
+                            region
+                                .assign_advice(|| "", config.c, offset, || table_row1)
+                                .expect("todo");
```

Comment:
What are these `todo`?? 
Is it that we have something pending? Otherwise we should choose a better message.

Reply:
Simply, replace `expect(...)` with `unwrap()`. https://github.com/privacy-scaling-explorations/halo2/pull/361/commits/081a70c199e52cb42780d54ee978c8b361294fb2
I believe it is okay because they are tests.